javascript-conditionals/javascript-conditionals-quiz-notes.md

```diff
@@ -0,0 +1,51 @@
+# javascript-conditionals-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Give 6 examples of comparison operators.
+
+- What data type do comparison expressions evaluate to?
+
+- What is the purpose of an `if` statement?
+
+- Is `else` required in order to use an `if` statement?
+
+- Describe the syntax (structure) of an `if` statement.
+
+- What are the three logical operators?
+
+- How do you compare two different expressions in the same condition?
+
+- What is the purpose of a `switch` statement?
+
+- Is a `default` clause required in order to use a `switch` statement?
```

Comment:
Don't forget to submit the quiz questions!

Reply:
Sorry, it looks like I didn't save it in VSC, but I took a screenshot above beforehand!